glambie/notebooks/demo-query-submitted-data.ipynb

```diff
@@ -0,0 +1,635 @@
bjhei697ZoPQC/Dvzv6PNthCAWeyk6JiIiIq2STodzjc11vrF439joaDgOK7tvLJtNpiPee29y/MqV6a2KZ8/C97+ftAN2dibVscZAtmHD8q5fRG5ICw1eu9z9xMyD7v6Yme1ayB2Y2UPALbNc9Ql3fyC6zScILYxfir9sltvP2htpZvcB9wHs2LFjIUsSERGRuVxv31ihkISxONiYTW9VXG6bNsHb3hYusXIZzp+fHsgeegi+/OXkNtu2XXvesR07ws9ARKRFFhq8cvNc17WQO3D3d893vZn9GvCLwLs82Xj2EtA4Kmg78Moc9/9Z4LMQ9ngtZE0iIiLShJnnG9uyJRyvVKaHsfHxZMS9+8ruG+vogNtvD5eYO1y6NH2q4pkzoV0xrujl8yGMNZ537ODBMEpeRGQRFjRcw8z+kjD84nMzjv8GcLe7/4slLcLsPcCfAD/r7pcbjh8B/oJkuMa3gf0ariEiIrLKzbZvrPF8Y+l0CHDZ7OqpLBWLYaz9zFH3o6PJbW67LamK7dwZqmuN0xllGg3XEEkstOL1YeCvzexXgePRsTcCHcAvtWAdnwE6gW9Z+J+wH7j7b7r7KTO7H3ia0IL4W9cLXSIiIrIKLHTf2NjY6tg3BpDLwZ13hkvMPZzweWZ17KGHwnUf/7iCl4gsSLPj5H8OuCP64yl3//u2rGqJVPESERFZI1b7vrG5TE7CiROwbx+89a0rvZpVSxUvkURTJ1B294eBh9u0FhEREbnRLGTfWNyqOD6enPx5pc83ls+HythynC9MRNaFpoKXiIiIyLKIzzfW0zP9fGOlUtiL1Xi+sTiMrcZ9YyIiEQUvERERWRvS6VBpyufn3jcWB7J431hcUVupfWMiIhEFLxEREVm7Zp5vDNbuvjERWdcUvERERGR9Wav7xkRkXVPwEhERkRuD9o2JyApS8BIREZEb11z7xsrlEMYa9425h0tcUYv3kYmILICCl4iIiEijVCqcTDmXm3/f2NgYZPRPKRFZGL1biIiIiFzPXPvGREQWSM3KIiIiIiIibabgJSIiIiIi0mYKXiIiIiIiIm2m4CUiIiIiItJmCl4iIiIiIiJtZu6+0mtoOTO7DFxYpm+3CbiyTN9LVie9Bm5sev5vbHr+Ra+B+e10980rvQiR1WBdBq/lZGaPufsbV3odsnL0Grix6fm/sen5F70GRGSh1GooIiIiIiLSZgpeIiIiIiIibabgtXSfXekFyIrTa+DGpuf/xqbnX/QaEJEF0R4vERERERGRNlPFS0REREREpM0UvGYws9vM7GEzO21mp8zsd6PjA2b2LTM7F33c2PA1HzOzZ83srJn9fMPxN5jZyei6/2JmthKPSZrT4tfAd6JjT0SXm1fiMcnCNfv8m9lN0e3HzewzM+5L7wFrTIuff/3+r0GLeA38UzM7Hv2uHzezdzbcl94DRGSKgte1qsBH3f124KeB3zKzw8C/A77t7vuBb0d/JrruV4AjwHuAPzWzdHRf/xW4D9gfXd6znA9EFq2VrwGAX3X3Y9Hl0nI+EFmUpp5/oAj8AfB7s9yX3gPWnlY+/6Df/7Wo2dfAFeC97n4n8GvAFxvuS+8BIjJFwWsGd7/o7o9Hn48Bp4FtwL3A56ObfR7459Hn9wJ/5e4ld38eeBZ4k5ltBTa4+6MeNtJ9oeFrZBVr1WtgWRctLdPs8+/uE+7+PcI/wKfoPWBtatXzL2vXIl4DP3H3V6Ljp4CcmXXqPUBEZlLwmoeZ7QJeB/wQ2OLuFyG8KQNxy8g24B8bvuyl6Ni26POZx2UNWeJrIPZnUZvRH6jNZG1Z4PM/F70HrHFLfP5j+v1fwxbxGvgA8BN3L6H3ABGZQcFrDmbWA3wF+LC7j85301mO+TzHZY1owWsAQpvRncDbo8u/bO0qpV2aeP7nvItZjuk9YI1owfMP+v1f05p9DZjZEeBTwL+OD81yM70HiNzAFLxmYWZZwpvtl9z9q9Hh16K2gbiFKO7Vfwm4reHLtwOvRMe3z3Jc1oAWvQZw95ejj2PAX6AWxDWhyed/LnoPWKNa9Pzr938Na/Y1YGbbgb8GPuTu56PDeg8QkWkUvGaIWkH+B3Da3f+k4ar/S9g0S/TxgYbjvxL1c+8mbJ79UdSGMGZmPx3d54cavkZWsVa9BswsY2abovvMAr8IPLUcj0EWbxHP/6z0HrA2ter51+//2tXsa8DM+oGvAx9z9+/HN9Z7gIjMpBMoz2BmbwO+C5wE6tHhjxP6u+8HdgAvAr/s7oPR13wC+HXCJKQPu/uD0fE3An8OdAEPAr/t+oGveq16DZhZN/APQBZIAw8BH3H32jI+HGnSIp//F4ANQAcwDNzt7k/rPWDtadXzD1xAv/9rUrOvATP7feBjwLmGu7nb3S/pPUBEGil4iYiIiIiItJlaDUVERERERNpMwUtERERERKTNFLxERERERETaTMFLRERERESkzRS8RERERERE2kzBS0RkGZjZTWb2RHR51cxejj4fN7M/Xen1iYiISHtpnLyIyDIzs08C4+7+xyu9FhEREVkeqniJiKwgM3uHmf1N9PknzezzZvZ3ZvaCmb3fzP6jmZ00s2+aWTa63RvM7BEzO25mf2tmW1f2UYiIiMj1KHiJiKwue4F7gHuB/wU87O53AgXgnih8fRr4oLu/AfifwB+t1GJFRERkYTIrvQAREZnmQXevmNlJIA18Mzp+EtgFHATuAL5lZkS3ubgC6xQREZEmKHiJiKwuJQB3r5tZxZONuHXCe7YBp9z9LSu1QBEREWmeWg1FRNaWs8BmM3sLgJllzezICq9JRERErkPBS0RkDXH3MvBB4FNm9iTwBPDWFV2UiIiIXJfGyYuIiIiIiLSZKl4iIiIiIiJtpuAlIiIiIiLSZgpeIiIiIiIibabgJSIiIiIi0mYKXiIiIiIiIm2m4CUiIiIiItJmCl4iIiIiIiJtpuAlIiIiIiLSZv8fnuOPHa/z0CkAAAAASUVORK5CYII=",
```

Comment:
We should avoid submitting outputs to git. Keeps file sizes down and makes reviews easier

Reply:
e.g. this is quite hard to review in its current format.